src/LIBRETRO/libretro.c

```diff
@@ -138,7 +138,7 @@ static void handle_key(uint8_t key, uint16_t retro_key)
          quasi88_key(key, 1);
          key_buffer[key] = true;
       }
-      else if (!key_on)
+      else if (key_buffer[key] && !key_on)
```

Comment:
If multiple RETRO keys were mapped to the same KEY88 key, and the last RETRO key was not being held down, it would cause the KEY88 key to never register being pressed because the first RETRO key would set the state to pressed, and the second RETRO key would set the state to not-pressed.